libdevice/cmake/modules/SYCLLibdevice.cmake

```diff
@@ -26,7 +26,8 @@ set(compile_opts
   )
 
 if (WIN32)
-  set(devicelib-obj-file ${binary_dir}/libsycl-msvc.o)
+  set(lib-suffix obj)
+  set(devicelib-obj-file ${obj_binary_dir}/libsycl-crt.${lib-suffix})
```

Comment:
We should be able to use the same add_custom_command for Linux and Windows now.

Reply:
The latest version used the same add_custom_command for libsycl-crt build on both Linux and Windows.